SwiftCompilerSources/Sources/Optimizer/InstructionSimplification/SimplifyConvertEscapeToNoEscape.swift

```diff
@@ -28,6 +28,9 @@ private extension ConvertEscapeToNoEscapeInst {
   ///   %3 = thin_to_thick_function %1 to $@noescape () -> ()
 
   func tryCombineWithThinToThickOperand(_ context: SimplifyContext) {
+    // compiling bridged.getFunctionTypeWithNoEscape crashes the 5.10 Windows compiler
```

Comment:
If this works on newer compilers, perhaps we should also include a version check?